component/admin/controllers/icalevent.php

```diff
@@ -1182,13 +1182,13 @@ private function doSave(& $msg)
 		if ($event = SaveIcalEvent::save($array, $this->queryModel, $rrule))
 		{
 			$row = new jIcalEventRepeat($event);
-			if (JEVHelper::canPublishEvent($row))
+			if (!JEVHelper::canPublishEvent($row) && !$event->state)
```

Comment:
That's exactly what happens there, right? IF (NO PUBLISHER and NOT PUBLISH) then we say event under review. Otherwise we say "Event saved"...